src/timed-event-manager.ts

```diff
@@ -20,17 +20,29 @@ interface TimedEvent extends EventBanner {
   name: string;
   eventType: EventType;
   shinyMultiplier?: number;
+  friendshipMultiplier?: number;
   startDate: Date;
   endDate: Date;
 }
 
 const timedEvents: TimedEvent[] = [
   {
     name: "Egg Skip Update",
-    eventType: EventType.GENERIC,
+    eventType: EventType.NO_TIMER_DISPLAY,
     startDate: new Date(Date.UTC(2024, 8, 8, 0)),
     endDate: new Date(Date.UTC(2024, 8, 12, 0)),
-    bannerKey: "egg-update",
+    bannerKey: "halloween2024-event-",
+    scale: 0.21,
+    availableLangs: [ "en", "de", "it", "fr", "ja", "ko", "es", "pt-BR", "zh-CN" ]
+  },
+  {
+    name: "Halloween Update",
+    eventType: EventType.SHINY,
+    shinyMultiplier: 2,
+    friendshipMultiplier: 2,
+    startDate: new Date(Date.UTC(2024, 9, 25, 0)),
+    endDate: new Date(Date.UTC(2024, 10, 4, 0)),
+    bannerKey: "halloween-banner",
```

Comment:
```suggestion
    bannerKey: "halloween2024-event-",
```
+ The next 2 lines can be removed. Don't know how to select multiple lines on mobile